### PR TITLE
tiny polish for DEFAULT_OPTIONS

### PR DESCRIPTION
**Pull request by landonwilkins**
no hard feels if you deny this pull request. just performing my morning code snoop.

love,
-me.

### CODE REVIEW

File: lib/rubocop/cop/team.rb [hunk 1]
```diff
@@ -15,11 +15,16 @@ class Team
       attr_reader :errors, :warnings, :updated_source_file
 
       alias updated_source_file? updated_source_file
+      
+      DEFAULT_OPTIONS = {
```
Comment (bbatsov) [inline lib/rubocop/cop/team.rb]: I'm move this right after the other constant.

**Comment by bbatsov** [comment]
Apart from my small remark you'll need to update the commit message to match our guidelines.

**Comment by rrosenblum** [comment]
The change makes sense to me.

**Comment by bbatsov** [comment]
@landonwilkins ping :-)

**Comment by landonwilkins** [comment]
Y'all are the bestest humans :heart:!

Sry for taking so long to get back to this!

Lemme know if ya want me to change anything.

kloveyoubai

Luck,

-me.

**Pull request merged**